pkg/device/device-util.go

```diff
@@ -121,6 +121,10 @@ func wipefsAndCreatePart(disk string, start uint64, partitionName string, size u
 		return err
 	}
 
+	if len(pList) == 0 {
+		return fmt.Errorf("could not find created partition %s", partitionName)
```

Comment:
return value of fmt.Errorf will be nil, we need to return error

Reply:
Nope. It returns error. Ref. https://golang.org/pkg/fmt/#Errorf